### ISSUE TITLE
Right after theme installation, "Activate" does not work

### ISSUE THREAD

**Issue by zieren** [getgrav/grav#3765]
Grav v1.7.43 - Admin v1.10.43

After installing a new theme via "Themes" -> "Add", I am taken to the theme details page, and there's an "Activate" button to activate the theme. This pops up a confirmation dialog, but confirming does nothing - the theme is not installed. When I navigate to the "Themes" top level page again and activate the new theme from there, it works fine.

This is reproducible for me with the latest Grav version and different themes.

**Comment by diomed** [comment]
that is true. i come across this often, but it''s really cache issue or something.

also, some themes are so outdated they wont work, 

cleaning cache ought to do it, tho it does seem a bit buggy it's not a big issue i suppose.

**Comment by pmoreno-rodriguez** [comment]
> Grav v1.7.43 - Admin v1.10.43
> 
> After installing a new theme via "Themes" -> "Add", I am taken to the theme details page, and there's an "Activate" button to activate the theme. This pops up a confirmation dialog, but confirming does nothing - the theme is not installed. When I navigate to the "Themes" top level page again and activate the new theme from there, it works fine.
> 
> This is reproducible for me with the latest Grav version and different themes.

Same for me.
